app/security.py

```diff
@@ -83,7 +83,6 @@ async def check_permission(request):
                     url=request.rel_url.human_repr(),
                     client_ip=request['client_ip'])
     except KeyError:
-        flash(request, VALIDATION_FAILURE_MSG)
```

Comment:
Still import for _flash_ and _VALIDATION_FAILURE_MSG_ but are now no longer used. Fails flake8 rules.